lib/fog/vsphere/models/compute/server.rb

```diff
@@ -289,10 +292,18 @@ def scsi_controllers
           attributes[:scsi_controllers] ||= service.list_vm_scsi_controllers(id)
         end
 
+        def nvme_controllers
+          attributes[:nvme_controllers] ||= service.list_vm_nvme_controllers(id)
+        end
+
         def scsi_controller
           scsi_controllers.first
         end
 
+        def nvme_controller
+          nvme_controllers.first
+        end
```

Comment:
From reading the above you're adding this, but deprecating it in the same PR. Wouldn't it be better to not introduce it at all?

Reply:
I suppose we can just not deprecate it. Scsi controllers were deprecated in https://github.com/fog/fog-vsphere/pull/54/commits/60a5dc320973ff6425bc941f24bae4e9187b57fa#diff-49106de6ade06895f1a509db7c9cd040ed76bfdb472548d8b99efc0468dce548R9 , not sure why
I tried to mirror the behavior but I guess we can undo that. @chris1984 , any idea why scsi_controllers were deprecated

Reply:
Note it's the singular `scsi_controller` that's deprecated. IMHO it makes sense if you can have more than one that there isn't a singular version. Since it was done in 2016, by this point you can probably delete it.